lib/rental.rb

```diff
@@ -0,0 +1,10 @@
+class Rental
+  def initialize(date, person, book)
+    @date = date
+    @person = person
+    @book = book
+  end
+
+  attr_accessor :date
+  attr_reader :person, :book
+end
```

Comment:
- [ ] As I mentioned in two of my comments about the associations, this is where you will add the `belongs-to` functionality, so basically you need to refactor your code a little bit to implement things properly. It seems like you missed the things here and this way is not going to work to add the rentals to the person or book as per requirement.

```suggestion
class Rental
  attr_accessor :date, :person, :book
  
  def initialize(date, person, book)
    @date = date
    @person = person
    @book = book
    book.rentals << self
    person.rentals << self
  end
end
```